packages/maskbook/src/env.d.ts

```diff
@@ -125,3 +125,8 @@ declare module '*.jpg' {
     const content: string
     export default content
 }
+
+declare module '*.svg' {
+    const content: any
```

Comment:
@Jack-Works we allow `any`?

Reply:
It can be string also, loading the actual svg still doesn't work currently.

Reply:
```ts
new URL('path', import.meta.url)
```

Reply:
https://github.com/DimensionDev/Maskbook/blob/643a95b9d3e3053a41cb5392008287fe5173c894/packages/maskbook/src/resources/image-payload/index.ts#L2-L10